examples/time/uniffi.toml

```diff
@@ -0,0 +1,9 @@
+[bindings.kotlin]
+package_name = "org.mozilla.uniffi.example.chronological"
+cdylib_name = "chronological"
+
+[bindings.python]
+cdylib_name = "chronological"
+
+[bindings.swift]
+cdylib_name = "chronological"
```

Comment:
Unless you need it for something, I think you should be able to get away without this `uniffi.toml` and just use the uniffi-provided defaults. Some of the existing examples use a `uniffi.toml` just so that we have some test coverage of how `uniffi.toml` works, but as mentioned above, I'm trying to move us away from that habit for new work.